apps/patient/src/components/ReadyText.tsx

```diff
@@ -0,0 +1,106 @@
+import dayjs, { Dayjs } from 'dayjs';
+import { Text } from '@chakra-ui/react';
+import { OrderFulfillment } from 'packages/sdk/dist/types';
+
+export type Maybe<T> = T | null;
```

Comment:
Presumably this exists elsewhere (e.g. in `packages/sdk/dist/types`). `Maybe<T>` should also be `T | null | undefined`

Reply:
imported